aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -199,8 +315,11 @@ impl AptosVM {
         }
     }
 
-    pub fn internals(&self) -> AptosVMInternals {
-        AptosVMInternals::new(&self.0)
+    pub fn version(&self) -> Result<Version, VMStatus> {
```

Comment:
This was/is very confusing: version only seems to be used for tests, and with unwrap, yet, alert has a quite scary message ("VM Startup failed"). Where do we even use this version, does it need to be Option, or should the error on None just be handled by the new() method?